src/CSV.jl

```diff
@@ -21,6 +21,8 @@ using WeakRefStrings
 export PosLenString, InlineString
 # CodecZlib is used for unzipping gzip files
 using CodecZlib
+# FilePathsBase allows more structured file path types
+using FilePathsBase
```

Comment:
Sorry, why does this need to be a direct dependency? I don't see reference to a `AbstractPath` type below or the `Path` constructor`